bazel/external/lightstep.sh

```diff
@@ -0,0 +1,20 @@
+# This is a Bazel genrule script. It acts like Bash, but has some extra
+# substitutions for locating dependencies.
+#
+# https://docs.bazel.build/versions/master/be/general.html#genrule
+
+configure="$${PWD}/$(location :configure)"
```

Comment:
Please either switch back to upper case or put in a PR first to adjust the entire repository to a lower case convention for internal variables. You can always re-lower case it later if you want to delay that repository conversion PR.

Reply:
Changed to upper-case. I don't want to barrier this PR on some giant shell blob cleanup.